pipeline/metadata/5.3.yaml

```diff
@@ -639,6 +639,22 @@
     - rgw
     - stage-2
 
+- name: "Testing extended RGW multisite bucket granular sync policy from primary"
+  suite: "suites/pacific/rgw/tier-1_rgw_ms-sync-policy-from-primary.yaml"
+  global-conf: "conf/pacific/rgw/rgw_multisite.yaml"
+  platform: "rhel-8"
+  rhbuild: "5.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - sanity
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-2
```

Comment:
Looks like for tag "sanity, tier-2, stage-2" we already have 5 suites, so might need to change the stage.
And also if this suite need to be executed as a part of 6.0 testing then changes required in 6.0 metadata file